src/Model/Delete/DeleteFilterPolicy.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace TractorCow\Fluent\Model\Delete;
+
+use InvalidArgumentException;
+use SilverStripe\ORM\DataObject;
+use TractorCow\Fluent\Extension\FluentFilteredExtension;
+use TractorCow\Fluent\Model\Locale;
+
+/**
+ * A policy that deletes the filtered rows.
+ *
+ * Requires {@see FluentFilteredExtension} on the object
+ */
+class DeleteFilterPolicy implements DeletePolicy
+{
+
+    /**
+     * @param DataObject|FluentFilteredExtension $record
+     * @return bool
```

Comment:
Since this operation doesn't return a boolean result of whether it was successful (which is what I might assume it does), it may be worth adding a note about what the boolean return is for

```suggestion
     * @return bool Whether this record is visible in any other locale after the delete operation
```